### PR TITLE
Hostile Humanoid Mobs Tweaks

### PR DESCRIPTION
**Pull request by ErhardSteinhauer**
### General Tweaks
- Fixed the issue with dungeons spawning multiple boss mobs.
- Reclassified some items as contraband: wizard items (hard suits, armor, staffs, and wands), blood cult items (armor, weapons), and expanded list of syndicate items considered contraband (jumpsuits, armor).
### Listening Post Bravo Changes
- Mapped 3 "low tier" syndicate agents (2 deckhands and 1 engineer). Added syndiCAT to the LP.
- Replaced regular docking airlocks with syndicate ones.
- Mapped vacuum markers.
### Bluespace Events Tweaks
- Syndicate Ship Interception, Blood Moon and WizFed Probe can now happen earlier in the round. Raised player number requirements for the events to trigger. Given how NPC-heavy those events are, they really should be triggered in the first half of the shift, before AI breaks.
### Syndicate Mobs Tweaks

_**Big changes:**_
- Static mob spawns are replaced with randomized ones. Now most mobs (primarily the ones that can be encountered in expeditions) have 4 variants with varied relative difficulty and loot: high danger/loot (10% chance to spawn), medium danger/loot (30%), low danger/loot (30%) and lame (30%). Only "high" and "medium" variants have guns in their loot. The rest have ammo and/or cash. Syndicate Commander in Expeditions is the only mob with static loadout now.
- The loot (apart from stuff that the mob is wearing) is stored in special bags, that can be worn as a backpack but don't function as one - it spawns a predefined set of items on use and disappears. Each randomized mob has a specific version of such a loot bag.

To expand on syndie mob variations, here is an example of Syndicate Captain spawner:
- High danger (10% chance to spawn): armed with plasteel sword and AP Python. Loot: AP Python, AP Speed Loader, 5k in cash.
- Medium danger (30% chance to spawn): armed with plasteel sword and Python. Loot: Python, x2 Speed Loaders, 2.5k in cash.
- Low danger (30% chance to spawn): armed with Python (no sword = no reflect). Loot: Speed Loader, 2.5k in cash.
- Lame (30% chance to spawn): armed with Python (no sword = no reflect). Loot: 2.5k in cash.

_**Small changes:**_
- Disabled climbing for mobs (they tend to trap themselves a lot with this behavior enabled).
- Added more comments to NPC-related yml files.
- Reduced ammo regeneration rate for Syndicate Operator and Commander (now they regenerate 1 bullet every 1.5 seconds).
- Replaced Syndicate Commander's hard suit with an unremovable version.
- Syndicate Operator should fire with less accuracy.
- Increased ammo regeneration rate for Syndicate medic (now they regenerate 1 dart every 2 seconds).
- Added a new set of organs that filter out chems from the entity's bloodstream faster.
- Marginally improved readability of spawner icons.
- Added syndicate mime. Fast melee specialist, armed with a chainsaw. The mob has a low chance of spawning in dungeons (less than 0.1%).
- Added syndicate chemwarrior. Sprays harmful chems on you and wears a new set of contraband armor: syndicate armored bio-suit. 4 variations of chems: Fresium, THC+Acid, Mute Toxin, and Weh Juice,  The mob has a low chance of spawning in dungeons (less than 0.1%).
- Added Syndicate Bomb Suit. It is contraband.
- Syndicate Grenadier now has 4 variants and has a low chance of spawning in dungeons (less than 0.1%).
- Increased the chance to reflect laser beams specifically.
- Incapacitated (critted) syndicate agents can be exchanged for FUC at NFSD Outpost.
- Syndicate mobs that are supposed to wield guns now have special gloves with "glued" guns to them. Suboptimal solution, but at least players will get some sort of visual clue of what they are up against.

### Wizard Federation Mobs Tweaks
- Disabled climbing for mobs (they tend to trap themselves a lot with this behavior enabled).
- Added minor health regen to Wizard Federation mobs to buff them a bit.
- Installed a new death acidifier, that deletes internal organs too.
- Mobs are now immune to anything that puts them to forced sleep.
- Conjured soap now deals 9 blunt and 10 stamina damage on impact. So it's 6 hits to slow, 10 hits to stun, 12 hits to kill unarmored target.
- Incapacitated (critted) wizards can be exchanged for FUC at NFSD Outpost.

### Blood Cult Mobs Tweaks
- Disabled climbing for mobs (they tend to trap themselves a lot with this behavior enabled).
- Mobs are now immune to anything that puts them to forced sleep.
- Ascended Cultist (expedition boss) can be butchered by players if they wish to obtain some of the cultist's powers for themselves: it drops an amulet, that allows them to summon a single Drained One periodically (friendly to NT personnel).
- Added minor health regen to Leech.
- Ranged mobs and the Leech now have a high chance to reflect laser beams.
- Nerfed damage (12 -> 7) on Drained Ones (Ascended Cultist summons).
- The cultist with the crossbow now has his quiver filled with bolts, the bag contains the crossbow and cash.
- Nerfed damage on cultists' crossbow bolts.
- Incapacitated (critted) cultists can be exchanged for FUC at NFSD Outpost.

## Why / Balance
Bug fixes and tweaks to hostile humanoids.

## Media
image
- [X] Changes to Listening Post Bravo

**Changelog**
:cl: erhardsteinhauer
- tweak: Prevents dungeons from spawning multiple bosses.
- tweak: Syndicate NPCs store loot in special bags, that can be worn as a backpack, but don't function as one - it spawns a predefined set of items on use and disappears.
- tweak: Ascended Cultist (blood cult expedition boss) can be butchered by players if they wish to obtain some of its powers for themselves.
- tweak: Listening Post Bravo is now manned by appropriate forces.
- tweak: Wizards and Blood Cultists have trouble sleeping.
- tweak: Incapacitated (put in crit) hostile agents can be exchanged for Frontier Uplink Coins at NFSD Outpost.
- tweak: Reclassified some items as contraband: wizard items (hard suits, armor, staffs, and wands), blood cult items (armor, weapons).

### CODE REVIEW

**Comment by dvir001** [comment]
Merge Conflict fixed

**Comment by radarseas** [comment]
it may be a good idea to replace the commander's suit from an elite syndicate hardsuit, currently people seem to be looting those a lot from low-risk expeds and getting end-game armor under 30 minutes into a round without much risk to themselves but cuffing a glitched mob or stripping it when the AI breaks.

**Comment by ErhardSteinhauer** [comment]
Reworked syndicate mobs.
- Static mob spawns replaced with randomized ones. Now most mobs (primarily the ones that can be encountered in expeditions) have 4 variants with varied relative difficulty and loot: high danger/loot (10% chance to spawn), medium danger/loot (30%), low danger/loot (30%) and lame (30%). Only "high" and "medium" variants have guns in their loot. The rest have ammo and cash. Syndicate Commander in expeditions is the only static mob now.
- The loot is stored in special bags, that can be worn as backpack but doesn't function as one - it spawns predefined set of items on use and disappears. Each randomized mob has a specific version of such loot bag.

So, for example, Syndicate Naval Captain NPC will have the following variations:
- High danger (10% chance to spawn): armed with plasteel sword and AP Python. Loot: AP Python, AP Speed Loader, 5k in cash.
- Medium danger (30% chance to spawn): armed with plasteel sword and Python. Loot: Python, x2 Speed Loaders, 2.5k in cash.
- Low danger (30% chance to spawn): armed with Python (no sword = no reflect). Loot: Speed Loader, 2.5k in cash.
- Lame (30% chance to spawn): armed with Python (no sword = no reflect). Loot: 2.5k in cash.

**Comment by ErhardSteinhauer** [comment]
Ascended Cultist (blood cult expedition boss) can be butchered by players if they wish to obtain some of its powers for themselves.

**Comment by ErhardSteinhauer** [comment]
Disabled climbing for syndicate/wizard/cultist mobs because they tend to trap themselves between objects a lot with this behavior enabled.

**Comment by ErhardSteinhauer** [comment]
Right, I believe this PR is in a decent enough state for a formal review (and I'm also fresh out of ideas for what else could be done with those mobs). I also updated PR description, so unless I forgot something it should be fairly accurate. I'd appreciate the feedback/requests, when and if you get a chance to review it, @Cheackraze, @MagnusCrowe. And I guess I'll ping you, @dvir001, just because I can.

**Comment by ErhardSteinhauer** [comment]
> Is there any counterplay to AP rounds + reflect mobs? Aside from being a zerg-force.

Yes.
0. Not getting hit.
1. Good armor, shields and cover.
2. Your reflect.
3. Violence of action.
4. Teamwork.

File: Resources/Prototypes/_NF/Entities/Objects/Weapons/Throwable/throwable_weapons.yml [hunk 1]
```diff
@@ -28,7 +29,24 @@
...
-    transferAmount: 4
+    transferAmount: 2
     blockSlots: NONE #tranquillizer darts shouldn't be blocked by a mask
   - type: InjectableSolution
     solution: ammo
+
+- type: entity # 4 hits to KO a spaceman
+  parent: DartSindicateImpedrezene
+  id: DartSindicateTranquilizer
+  name: Syndicate Tranquilizer Dart
```
Comment (dvir001) [inline Resources/Prototypes/_NF/Entities/Objects/Weapons/Throwable/throwable_weapons.yml:40]: Why capital in names reeee

Comment (ErhardSteinhauer) [reply Resources/Prototypes/_NF/Entities/Objects/Weapons/Throwable/throwable_weapons.yml:40]: ugh, I'll fix it

File: Resources/Prototypes/_NF/Entities/Mobs/NPCs/base_humanoid_hostile.yml [hunk 2]
```diff
@@ -55,10 +86,15 @@
...
-        true
+        false
   - type: NpcFactionMember
     factions:
-      - SimpleNeutral
+      - SimpleHostile
+  - type: Carriable # Carrying system from nyanotrasen.
+  - type: MobPrice
+    price: 1500 # Kidnapping a living person and selling them for cred is a good move.
+    deathPenalty: 0.5 # However they really ought to be living and intact, otherwise they're worth 2x less.
```
Comment (dvir001) [inline Resources/Prototypes/_NF/Entities/Mobs/NPCs/base_humanoid_hostile.yml:96]: Real.

Comment (dvir001) [reply Resources/Prototypes/_NF/Entities/Mobs/NPCs/base_humanoid_hostile.yml:96]: Also you cannot even sell alive mobs unless they are sold on a ship, with the ship.

Comment (ErhardSteinhauer) [reply Resources/Prototypes/_NF/Entities/Mobs/NPCs/base_humanoid_hostile.yml:96]: You can sell mob in crit though: I did sell critted syndies for FUCs.

Comment (ErhardSteinhauer) [reply Resources/Prototypes/_NF/Entities/Mobs/NPCs/base_humanoid_hostile.yml:96]: Not sure about hard cash though, haven't tried. 1500 ain't worth the hussle.

**Comment by Dysh3s** [comment]
> > Is there any counterplay to AP rounds + reflect mobs? Aside from being a zerg-force.
> 
> Yes. 0. Not getting hit.
> 
>     1. Good armor, shields and cover.
> 
>     2. Your reflect.
> 
>     3. Violence of action.
> 
>     4. Teamwork.

0. AI rush you and are very precise. Not to mention that most of the time you are slower than them. Not getting hit is a miracle scenario.
1. AP ammo ignores armor and shields though. It's a 25 damage per hit, that you can't mitigate. Cover - maybe, but again - AI rush you.
2. ESwords don't have reflect in frontier. Only DESword does.
3. Oh your lads are violent all right.
4. Well, aside from being a zerg force.

The only counterplay I see as of now is a pre-injecting puncturase before heading in. Even then it'll help you mitigate one shot max. And it's cheesy as hell.

**Comment by Dysh3s** [comment]
See, that's where my issue comes in. He can kill you in two seconds from screen range, but you have to magdump kammerer in his face, while also receiving some of the shells back due to plastitanium sword (does he drop it though?). It feels like there is no actual counterplay, aside from being in group.

**Comment by dvir001** [comment]
> aside from being in group.

Yes.

**Comment by ErhardSteinhauer** [comment]
> See, that's where my issue comes in. He can kill you in two seconds from screen range, but you have to magdump kammerer in his face, while also receiving some of the shells back due to plastitanium sword (does he drop it though?). It feels like there is no actual counterplay, aside from being in group.

Still, it is possible to kill that NPC solo. You gonna get hurt, sure. Or you might die. imo games aren't fun without a bit of danger. Besides the version with AP rounds has only 10% spawn chance and it does drop plasteel sword. If you're truing to kill that mob solo, reflect is the best counter play, but teamplay is better. Teamplay with reflect even better.

(tbh I think that I might've nerfed syndie mobs too much, we'll see how it goes when/if this PR gets submitted I guess)

**Review by Cheackraze — APPROVED** [review]
making the guns into just gloves sprites is hilarious

**Comment by Qiler** [comment]
Syndies dont seem to go into crit anymore. They always acidify. With unequipable armor change, Option to sell critted baddies for FUK and addition of loot backpacks it seems unintended. The only ways to get loot backpacks right now is to strip them with abuse of game mechanics, mob them with stun battons to cuff them or when AI brakes.

**Pull request merged**